Keas.Mvc/Services/IdentityService.cs

```diff
@@ -247,12 +248,18 @@ public async Task<PPSDepartmentResult> GetPpsDepartment(string ppsCode)
         public async Task<string> GetTitle(string iamId)
         {
             var title = string.Empty;
-            var clientws = new IetClient(_authSettings.IamKey);
-            var result = await clientws.PPSAssociations.Search(PPSAssociationsSearchField.iamId, iamId);
-            if (result.ResponseData.Results.Length > 0)
+            try
             {
-
-                title = result.ResponseData.Results.FirstOrDefault(a => !string.IsNullOrWhiteSpace(a.titleOfficialName))?.titleOfficialName;
+                var clientws = new IetClient(_authSettings.IamKey);
+                var result = await clientws.PPSAssociations.Search(PPSAssociationsSearchField.iamId, iamId);
+                if (result.ResponseData.Results.Length > 0)
+                {
+                    title = result.ResponseData.Results.FirstOrDefault(a => !string.IsNullOrWhiteSpace(a.titleOfficialName))?.titleOfficialName;
+                }
+            }
+            catch (Exception ex)
+            {
+                Log.Error($"Getting Title for IamId {iamId}. Exception: {ex.Message} -- {ex.InnerException.Message} ");
```

Comment:
@jSylvestre isn't there a version of `Log.Error` which takes in the actual error instead of just the message?  That way you get a stack trace and other more useful info than writing the exception out into a string.